### ISSUE TITLE
Introspection breaks when the primary key is defined on multiple columns 

### ISSUE THREAD

**Issue by ridem** [drizzle-team/drizzle-kit-mirror#12]
- Command: `drizzle-kit introspect:pg`
- PostgreSQL 12

Error:
```
/.../node_modules/drizzle-kit/index.js:26434
                  name: tableCompositePkName.rows[0].primary_key,
                                                     ^

TypeError: Cannot read properties of undefined (reading 'primary_key')
    at /.../node_modules/drizzle-kit/index.js:26434:54
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.14.0
```

In order to get the introspection running, I've had to replace this bit in drizzle-kit:

```ts
                const tableCompositePkName = await db.query(
                  `SELECT conname AS primary_key
              FROM   pg_constraint 
              WHERE  contype = 'p' 
              AND    connamespace = 'public'::regnamespace  
              AND    conrelid::regclass::text = $1;`,
                  [tableName]
                );
```

with something like this, that accounts for the fact that the names in `conrelid::regclass::text` are surrounded with quotes:

```ts
                const tableCompositePkName = await db.query(
                  `SELECT conname AS primary_key
              FROM   pg_constraint 
              WHERE  contype = 'p' 
              AND    connamespace = 'public'::regnamespace  
              AND    conrelid::regclass::text = $1;`,
                  [`"${tableName}"`]
                );
```

**Comment by AndriiSherman** [comment]
Thanks @ridem!
Will send you internal release with fixes today, so you could check it

**Comment by AndriiSherman** [comment]
@ridem have some update
Just tested this part for both `postgres:12` and `postgres:14` and no issues with simple `[tableName]`, but faced same as you with ```[`"${tableName}"`]```. So it's "reverse" behaviour
Need a bit more time to investigate the reason and think if I need some extra info from you

---
Maybe something like this will be universal solution, but need to test more
```typescript
await db.query(
              `SELECT conname AS primary_key
              FROM   pg_constraint 
              WHERE  contype = 'p' 
              AND    connamespace = 'public'::regnamespace  
              AND   (conrelid::regclass::text = $1 OR conrelid::regclass::text = $2);`,
              [`"${tableName}"`, tableName]
)
```

**Comment by ridem** [comment]
It apparently only depends on the actual table name, so your suggested fix will work.

The name will be surrounded with quotes if it contains characters outside `[a-z0-9_]` or if it's a [reserved word](https://www.postgresql.org/docs/current/sql-keywords-appendix.html) (like `user`)


<table>
<tr>
	<td>

```sql
SELECT
	tablename
FROM
	pg_catalog.pg_tables
WHERE
	schemaname != 'pg_catalog'
	AND schemaname != 'information_schema';
```

</td>
	<td>

```
organization
user
Organization
users
user_role
table:name
thing_1
```

</td>
<tr>
	<td>

```sql
SELECT
	conrelid::regclass::text
FROM
	pg_constraint
WHERE
	contype = 'p'
	AND connamespace = 'public'::regnamespace
```

</td>
	<td>

```
organization
"user"
"Organization"
users
user_role
"table:name"
thing_1
```

</td>
</table>

**Comment by AndriiSherman** [comment]
@ridem yeah, make sense!
I'll add this fix to one of next releases and will mention you in release notes
Thanks a lot!

**Comment by AndriiSherman** [comment]
@ridem fixed in `drizzle-kit@0.17.4` 🎉
